packages/ai/core/generate-object/generate-object.ts

```diff
@@ -272,57 +148,8 @@ functionality that can be fully encapsulated in the provider.
         currentDate?: () => Date;
       };
     },
-): Promise<GenerateObjectResult<ENUM>>;
-/**
-Generate JSON with any schema for a given prompt using a language model.
-
-This function does not stream the output. If you want to stream the output, use `streamObject` instead.
-
-@returns
-A result object that contains the generated object, the finish reason, the token usage, and additional information.
- */
-export async function generateObject(
-  options: Omit<CallSettings, 'stopSequences'> &
-    Prompt & {
-      output: 'no-schema';
-
-      /**
-The language model to use.
-     */
-      model: LanguageModel;
-
-      /**
-The mode to use for object generation. Must be "json" for no-schema output.
-     */
-      mode?: 'json';
+): Promise<GenerateObjectResult<TYPE>>;
 
-      /**
-A function that attempts to repair the raw output of the mode
-to enable JSON parsing.
-     */
-      experimental_repairText?: RepairTextFunction;
-
-      /**
-Optional telemetry configuration (experimental).
-       */
-      experimental_telemetry?: TelemetrySettings;
-
-      /**
-Additional provider-specific options. They are passed through
-to the provider from the AI SDK and enable provider-specific
-functionality that can be fully encapsulated in the provider.
- */
-      providerOptions?: ProviderOptions;
-
-      /**
-       * Internal. For test use only. May change without notice.
-       */
-      _internal?: {
-        generateId?: () => string;
-        currentDate?: () => Date;
-      };
-    },
-): Promise<GenerateObjectResult<JSONValue>>;
```

Comment:
now it returns `any`:
![CleanShot 2025-04-11 at 15 38 59](https://github.com/user-attachments/assets/d9f13de3-a397-4735-bb07-1bf43a084e86)

Reply:
fixed!